seed/analysis_pipelines/co2.py

```diff
@@ -418,7 +418,8 @@ def _run_analysis(self, meter_readings_by_analysis_property_view, analysis_id):
         analysis_property_view.parsed_results = {
             'Average Annual CO2 (kgCO2e)': co2['average_annual_kgco2e'],
             'Annual Coverage %': co2['annual_coverage_percent'],
-            'Total Annual Meter Reading (MWh)': co2['total_annual_electricity_mwh']
+            'Total Annual Meter Reading (MWh)': co2['total_annual_electricity_mwh'],
+            'Total GHG Emissions Intensity (kgCO2e/ft\u00b2/year)': co2['average_annual_kgco2e'] / property_view.state.gross_floor_area.magnitude
```

Comment:
The unit is hardcoded in the model so I hardcoded it here. However, this could be done dynamically by substituting the unit string for the following variable.
```
total_ghg_emission_intensity_units = format(property_view.state.total_ghg_emissions_intensity.u, '~P')
```